packages/templates/wasm/rust/meta/test.graphql

```diff
@@ -0,0 +1,5 @@
+query {
```

Comment:
I strongly think we shouldn't keep this in the templates since we deprecated the graphql queries and also thinking about replacing it completely in the future.

Reply:
okay, i'll remove it

Reply:
done